tests/Feature/RegisteredUserTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+
+class RegisteredUserTest extends TestCase
+{
+
+    use RefreshDatabase;
+
+    /** @test */
+    public function a_user_can_register()
+    {
+        $user = [
+            'name' => 'Jaydel',
+            'email' => 'testemail@test.com',
+            'password' => 'passwordtest',
+            'password_confirmation' => 'passwordtest'
+        ];
+
+        $response = $this->post('/register', $user);
+
+        $response->assertRedirect('/');
+
+        $this->assertDatabaseHas('users', [
+            'name' => 'Jaydel',
+            'email' => 'testemail@test.com',
+        ]);
+    }
+
+
+    /** @test */
+    public function only_a_registered_user_can_see_content()
+    {
+        $user = User::factory()->create();
+
+        $this->actingAs($user)
+            ->getJson('/home')
```

Comment:
Even if the user is not registered, I believe the user should be able to see the homepage right? So after the front view has been removed, this tests could also be removed ;D

Reply:
I actually deleted this file Vick as we are keeping Kieran's test cases, I hope that's ok?